src/test/java/com/google/api/codegen/testdata/java_main_library.baseline

```diff
@@ -153,7 +153,6 @@ import javax.annotation.Generated;
  * </pre>
  */
 @Generated("by GAPIC")
-@ExperimentalApi
```

Comment:
I didn't want to create two whole new services (for ALPHA and GA), that would be a bit of overhead.